src/inspector_socket.cc

```diff
@@ -445,9 +445,10 @@ static int header_value_cb(http_parser* parser, const char* at, size_t length) {
   struct http_parsing_state_s* state = (struct http_parsing_state_s*)
       (reinterpret_cast<inspector_socket_t*>(parser->data))->http_parsing_state;
   state->parsing_value = true;
-  if (state->current_header && strncmp(state->current_header,
-                                       SEC_WEBSOCKET_KEY_HEADER,
-                                       sizeof(SEC_WEBSOCKET_KEY_HEADER)) == 0) {
+  if (state->current_header &&
+      strncasecmp(state->current_header,
+      SEC_WEBSOCKET_KEY_HEADER,
+      sizeof(SEC_WEBSOCKET_KEY_HEADER)) == 0) {
```

Comment:
Can you add a version of `StringEqualNoCase()` that takes a size parameter and use that?

Style: arguments should line up here.